.rubocop.yml

```diff
@@ -73,9 +73,6 @@ Style/Documentation:
 Style/ClassAndModuleChildren:
   Enabled: false
 
-Style/TrailingComma:
-  Enabled: false
```

Comment:
The style violations in existing code are because of this change to the rubocop style, which shouldn't be needed for this PR.